app/src/main/java/com/android/sample/mlUtils/exercisesCriterions/AngleCriterionComments.kt

```diff
@@ -1,39 +1,167 @@
+              PoseDetectionJoints.RIGHT_ELBOW_SHOULDER_OPPSHOULDER,
+          )),
+  RIGHT_ELBOW_SHOULDER_HIP_NOT_RIGHT(
+      "Keep your right arm straight parallel to the ground",
+      focusedJoints =
+          listOf(
+              PoseDetectionJoints.LEFT_ELBOW_SHOULDER_HIP,
+              PoseDetectionJoints.RIGHT_ELBOW_SHOULDER_HIP,
+          )),
+  BOTH_ELBOW_SHOULDER_HIP_NOT_RIGHT(
+      "Keep your arms straight parallel to the ground",
+      focusedJoints =
+          listOf(
+              PoseDetectionJoints.LEFT_ELBOW_SHOULDER_HIP,
+              PoseDetectionJoints.RIGHT_ELBOW_SHOULDER_HIP,
+          )),
+  ELBOW_SHOULDER_HIP_NOT_FLAT(
+      "Put your arms aligned with your back.",
+      focusedJoints =
+          listOf(
+              PoseDetectionJoints.LEFT_ELBOW_SHOULDER_HIP,
+              PoseDetectionJoints.RIGHT_ELBOW_SHOULDER_HIP,
+          )),
+  LEFT_HIP_KNEE_ANKLE_NOT_FLAT(
+      "Keep your left leg stretched",
+      focusedJoints = listOf(PoseDetectionJoints.LEFT_HIP_KNEE_ANKLE)),
+  RIGHT_HIP_KNEE_ANKLE_NOT_FLAT(
+      "Keep your right leg stretched",
+      focusedJoints = listOf(PoseDetectionJoints.RIGHT_HIP_KNEE_ANKLE)),
+  BOTH_HIP_KNEE_ANKLE_NOT_FLAT(
+      "Keep your legs stretched",
+      focusedJoints =
+          listOf(
+              PoseDetectionJoints.LEFT_HIP_KNEE_ANKLE, PoseDetectionJoints.RIGHT_HIP_KNEE_ANKLE)),
+  RIGHT_HIP_KNEE_ANKLE_NOT_RIGHT(
+      "Keep an angle of 90 degree with your right knee.",
+      focusedJoints = listOf(PoseDetectionJoints.RIGHT_HIP_KNEE_ANKLE)),
+  LEFT_HIP_KNEE_ANKLE_NOT_RIGHT(
+      "Keep an angle of 90 degree with your left knee.",
+      focusedJoints = listOf(PoseDetectionJoints.LEFT_HIP_KNEE_ANKLE)),
+  BOTH_HIP_KNEE_ANKLE_NOT_RIGHT(
+      "Keep an angle of 90 degree with your knees.",
+      focusedJoints =
+          listOf(
+              PoseDetectionJoints.LEFT_HIP_KNEE_ANKLE, PoseDetectionJoints.RIGHT_HIP_KNEE_ANKLE)),
+  RIGHT_OPPHIP_HIP_KNEE_NOT_OPEN(
+      "Spread more your right leg.",
+      focusedJoints = listOf(PoseDetectionJoints.RIGHT_OPPHIP_HIP_KNEE)),
+  LEFT_OPPHIP_HIP_KNEE_NOT_OPEN(
+      "Spread more your left leg.",
+      focusedJoints = listOf(PoseDetectionJoints.LEFT_OPPHIP_HIP_KNEE)),
+  BOTH_OPPHIP_HIP_KNEE_NOT_OPEN(
+      "Spread more your legs.",
+      focusedJoints =
+          listOf(
+              PoseDetectionJoints.LEFT_OPPHIP_HIP_KNEE, PoseDetectionJoints.RIGHT_OPPHIP_HIP_KNEE)),
+  RIGHT_OPPHIP_HIP_KNEE_NOT_CLOSED(
+      "Tighten more your right leg.",
+      focusedJoints = listOf(PoseDetectionJoints.RIGHT_OPPHIP_HIP_KNEE)),
+  LEFT_OPPHIP_HIP_KNEE_NOT_CLOSED(
+      "Tighten more your left leg.",
+      focusedJoints = listOf(PoseDetectionJoints.LEFT_OPPHIP_HIP_KNEE)),
+  BOTH_OPPHIP_HIP_KNEE_NOT_CLOSED(
+      "Tighten more your legs",
+      focusedJoints =
+          listOf(
+              PoseDetectionJoints.LEFT_OPPHIP_HIP_KNEE, PoseDetectionJoints.RIGHT_OPPHIP_HIP_KNEE))
```

Comment:
Put a period at the end of each sentence, or not, but don't mix the two.

I'd vote for a period.